modules/app/controllers/webhook.js

```diff
@@ -21,10 +21,27 @@ const FAILED_REASON = {
 }
 
 const CURRENCIES = {
-  brl: 'R$',
-  usd: '$',
+  aud: '$',
   eur: '€',
-  dkk: 'DK'
```

Comment:
You should not remove DK, it's a currency supported too.

Reply:
In have just moved it a few lines below